fineract-core/src/main/java/org/apache/fineract/infrastructure/core/service/database/JdbcJavaType.java

```diff
@@ -136,11 +136,19 @@ public static JdbcJavaType getByTypeName(@NotNull DatabaseType dialect, String n
         for (JdbcJavaType type : values()) {
             DialectType dialectType = type.getDialectType(dialect);
             if (dialectType.getNameResolved().equals(name)) {
+                // NOTE: make MySQL systems happy aka TINYINT vs BOOLEAN issue!
+                if (type.canBooleanType(dialect)) {
+                    return BOOLEAN;
+                }
                 return type;
             }
             if (dialectType.alterNames != null) {
                 for (String alterName : dialectType.alterNames) {
                     if (alterName.equals(name)) {
+                        // NOTE: make MySQL systems happy aka TINYINT vs BOOLEAN issue!
```

Comment:
Can you please explain the use-case why do you need this and why you do not change the BOOLEAN and TINYINT behaviour for mysql?

Reply:
Have to do the release. Please propose a pull request if you see any need for further improvement. As is works.